test/docs/TEST.ROOT

```diff
@@ -0,0 +1,28 @@
+# This file identifies the root of the test-suite hierarchy.
```

Comment:
This file should have a standard copyright header, too. I see now that the existing TEST.ROOT in jdk, langtools and jaxp is also missing this header. If you want, you can add it to them in this PR too, or we'll open a separate issue for that. But in any case this new file needs a proper header.